src/app/modules/core/inbox/inbox.component.scss

```diff
@@ -0,0 +1,145 @@
+$width280: 280px;
```

Comment:
color, all varaibles  should be defined in global scss file. not in each ocmponent file